### DISCUSSION TITLE
New installation v6 - strtoupper(): Argument #1 ($string) must be of type string, array given

### DISCUSSION THREAD

**Discussion by stefanogriggio** [Laravel-Backpack/community-forum#644] [Q&A (Help)] [answered]
I have a working v5 installation with Laravel 8. Rather than upgrading to v6, I've created a new Laravel 10 project and installed v6 with the intention of recreating all my models. Loading the admin page I obtain

App \ Http \ Middleware \ RedirectIfAuthenticated : 28
strtoupper(): Argument #1 ($string) must be of type string, array given

Any help?

Thanks

**Comment by pxpm** [comment]
It seems to me that the error is related with arrays in the field names. You should be using `name => [name1, name2]` (as an array), and we have deprecated the array names in favor of comma separated ones: `name => name1,name2`. 

Even creating a new app and adding your old code there, you should follow the https://backpackforlaravel.com/docs/6.x/upgrade-guide as some steps require changes in your previous code. 

You can ignore some steps as technically you are a new installation, but most of them you still need to follow  to have your old files working in a new application. 

Also note that if we are talking about blade files overwritten too I would also advise you to do the upgrade using `coreui-v2` theme.

Cheers

**Comment by stefanogriggio** [comment, marked as answer]
Sorry Pedro.
I missed an error during theme installation : "Your requirements could not be resolved to an installable set of packages."
Solved.

Thanks
